specification/compute/resource-manager/readme.md

```diff
@@ -27,17 +27,30 @@ To see additional help and options, run:
 
 
 
-### Basic Information 
+### Basic Information
 These are the global settings for the Compute API.
 
 ``` yaml
 title: ComputeManagementClient
 description: Compute Client
 openapi-type: arm
-tag: package-2017-03
+tag: package-2017-09

```

Comment:
This README is missing tag `package-2017-09` and tag `package-container-service-2017-09`

Reply:
I noticed as soon as I said it was "ready." :-( Updated, thanks @lmazuel.